python/pyarrow/tests/test_ipc.py

```diff
@@ -161,14 +164,17 @@ def test_empty_file():
         pa.ipc.open_file(pa.BufferReader(buf))
 
 
+@pytest.mark.numpy
 def test_file_simple_roundtrip(file_fixture):
```

Comment:
Another to do: update `IpcFixture.write_batches` to not use numpy (that would avoid a lot of marks in this file, I think)